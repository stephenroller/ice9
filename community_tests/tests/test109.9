# fa i := 0 to 10 -> break; af								succeed

fa i := 0 to 10 -> 
	write i;
	break; 
af
