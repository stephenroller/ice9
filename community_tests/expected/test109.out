Loading file: generated/test109.tm
0 
Number of instructions executed = 32
